### PR TITLE
Add some information about how to test master

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

**Comment by geoand** [comment]
Isn't this fairly similar to what we already have at https://github.com/quarkusio/quarkus/blob/82bbfae8ccece40815f5feded3bbcc6d1664c1f2/CONTRIBUTING.md#usage?

**Comment by gsmet** [comment]
Ah yes, looks like it. But it's a bit buried and too chatty for my taste when someone just wants to quickly check master.

**Comment by geoand** [comment]
Understood

**Review by geoand — APPROVED** [review]

**Comment by gsmet** [comment]
I will tweak it a bit and add some links to the other section.

**Comment by geoand** [comment]
Cool, let me know when you want a new review

**Comment by gsmet** [comment]
@geoand I gave it another shot. Also added some guidelines as requested a while ago by Emmanuel.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,8 +174,23 @@ After the build was successful, the artifacts are available in your local Maven
 
 To include them into your project a few things have to be changed.
 
-#### With Gradle
+#### With Maven
```
Comment (geoand) [inline CONTRIBUTING.md:177]: I like how Maven if first now :)

**Review by geoand — APPROVED** [review]
Clarifies things quite a bit!

**Comment by famod** [comment]
`-T` > 1 for Maven might increase developer joy. :wink:

**Pull request merged**